Movie/Movie/Core/API/URL.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  URL.swift
+//  Movie
+//
+//  Created by Дмитрий Супрун on 26.10.22.
+//
+
+import Foundation
+/// URLS
+enum Url {
```

Comment:
```suggestion

/// URLS
enum MovieUrl {
```
системные имена лучше не использовать

Reply:
Renamed